app/services/container_topology_service.rb

```diff
@@ -80,10 +80,10 @@ def entity_display_type(entity)
           "Container" # the container entity itself
         end
       else
-        if name.eql? "Vm"
-          'VM'
+        if entity.kind_of?(Vm)
+          name.upcase # turn Vm to VM because it's an abbreviation
```

Comment:
I realize this change is better than before but I wasn't aware of the original code.

Is this for display purposes?  I'm surprised we need to treat Vm differently.  If we want to deal with properly capitalizing abbreviations, it sounds like a generic problem that should be done for all display areas and not specific to this code.